src/ui/public/query_bar/directive/suggestion.html

```diff
@@ -0,0 +1,23 @@
+<div class="suggestionItem suggestionItem--{{item.type}}">
+  <div class="suggestionItem__type">
+    <div ng-switch="item.type">
+      <div ng-switch-when="field">
+        <icon type="'kqlField'"></icon>
+      </div>
+      <div ng-switch-when="value">
+        <icon type="'kqlValue'"></icon>
+      </div>
+      <div ng-switch-when="recentSearch">
+        <icon type="'search'"></icon>
+      </div>
+      <div ng-switch-when="conjunction">
+        <icon type="'kqlSelector'"></icon>
+      </div>
+      <div ng-switch-when="operator">
+        <icon type="'kqlOperand'"></icon>
+      </div>
+    </div>
+  </div>
+  <div class="suggestionItem__text" ng-bind-html="item.text"></div>
+  <div class="suggestionItem__description" ng-if="item.type !== 'recentSearch'" ng-bind-html="item.description"></div>
```

Comment:
Why do we need to use `ng-bind-html` on either of the two lines above? Wouldn't `{{}}` work? `ng-bind-html` will sanitize the value so fields like `xss` in makelogs will show an empty string as a suggestion. `{{}}` will escape the html characters so the field value still displays correctly.

Not 100% sure but it also seemed like switching to `{{}}` fixed the console errors I mentioned in my summary comment.

Reply:
I see now that the suggestion descriptions can have html in them. But won't this cause issues if the field names and/or values contain html characters? It seems like we either need to html escape all user provided values in the suggestion modules themselves before returning the suggestion objects, or refactor the suggestion objects so they don't need to contain html.